packages/compass-aggregations/src/components/saving-pipeline-modal/saving-pipeline-modal.jsx

```diff
@@ -62,13 +62,12 @@ class SavingPipelineModal extends PureComponent {
         trackingId="save_pipeline_modal"
         data-testid="save-pipeline-modal"
       >
-        <input
```

Comment:
Drive-by: This wasn't using ModalInput or TextInput before. Only case of this anywhere I could find. Since it is a near-identical aggregations-related modal to the others I changed it with them to work the same.